lib/rucio/web/ui/static/account_rse_usage.js

```diff
@@ -10,22 +10,25 @@
  * - Cedric Serfon, <cedric.serfon@cern.ch>, 2019
  */
 
-var result_html = '<table id="resulttable" class="compact stripe order-column" style="word-wrap: break-word;"><thead><tr><th>RSE</th><th>Quota</th><th>Used</th><th>Available</th><th>Files</th></tr></thead><tfoot><tr><th>RSE</th><th>Quota</th><th>Used</th><th>Available</th><th>Files</th></tr></tfoot></table>'
+var result_local_html = '<table id="resulttable_local" class="compact stripe order-column" style="word-wrap: break-word;"><thead><tr><th>RSE</th><th>Quota</th><th>Used</th><th>Available</th><th>Files</th></tr></thead><tfoot><tr><th>RSE</th><th>Quota</th><th>Used</th><th>Available</th><th>Files</th></tr></tfoot></table>'
+var result_global_html = '<table id="resulttable_global" class="compact stripe order-column" style="word-wrap: break-word;"><thead><tr><th>RSE Expression</th><th>Quota</th><th>Used</th><th>Available</th><th>Files</th></tr></thead></table>'
```

Comment:
Nit and out of scope, but I'd be in favour of renaming these `resulttable_*` to `result_table_*` for better readability and for consistency with the fact that we are now using the underscore to separate the last word anyway